packages/cra-template-imodeljs/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@bentley/cra-template-imodeljs",
+  "version": "1.0.0",
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "engines": {
+    "node": ">=10"
+  },
+  "main": "template.json",
+  "description": "A quick start Create React App template for imodeljs",
+  "keywords": [
+    "react",
+    "create-react-app",
+    "cra-template",
+    "template",
+    "imodeljs",
+    "bentley",
+    "typescript"
+  ],
+  "files": [
+    "template",
+    "template.json"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/create-react-app.git",
```

Comment:
repositiory.url and bugs.url incorrect

Reply:
Thanks for pointing out. Now the repository.url is changed to https://github.com/imodeljs/create-react-app.git and bugs.url is changed to https://github.com/imodeljs/create-react-app/issues.